test/suites/scale/deprovisioning_test.go

```diff
@@ -0,0 +1,631 @@
+			// the other provisioners
+			driftNodeTemplate := nodeTemplate.DeepCopy()
+			driftNodeTemplate.Name = test.RandomName()
+			provisionerMap[driftValue].Spec.ProviderRef.Name = driftNodeTemplate.Name
+
+			env.MeasureDurationFor(func() {
+				By("kicking off provisioning by applying the provisioner and nodeTemplate")
+				env.ExpectCreated(driftNodeTemplate, nodeTemplate)
+				for _, p := range provisionerMap {
+					env.ExpectCreated(p)
+				}
+
+				env.EventuallyExpectCreatedMachineCount("==", expectedNodeCount)
+				env.EventuallyExpectCreatedNodeCount("==", expectedNodeCount)
+				env.EventuallyExpectInitializedNodeCount("==", expectedNodeCount)
+
+				// Wait for all pods across all deployments we have created to be in a healthy state
+				wg = sync.WaitGroup{}
+				for _, d := range deploymentMap {
+					wg.Add(1)
+					go func(dep *appsv1.Deployment) {
+						defer GinkgoRecover()
+						defer wg.Done()
+
+						env.EventuallyExpectHealthyPodCount(labels.SelectorFromSet(dep.Spec.Selector.MatchLabels), int(lo.FromPtr(dep.Spec.Replicas)))
+					}(d)
+				}
+				wg.Wait()
+			}, aws.ProvisioningEventType, multipleDeprovisionersTestGroup, defaultTestName)
+
+			env.Monitor.Reset() // Reset the monitor so that we now track the nodes starting at this point in time
+
+			By("scaling down replicas across deployments")
+			deploymentMap[consolidationValue].Spec.Replicas = lo.ToPtr[int32](int32(int(float64(replicas) * 0.2)))
+			deploymentMap[emptinessValue].Spec.Replicas = lo.ToPtr[int32](0)
+			for _, d := range deploymentMap {
+				env.ExpectUpdated(d)
+			}
+
+			env.MeasureDurationFor(func() {
+				By("enabling deprovisioning across provisioners")
+				// Create provisioners for expiration and drift so that these provisioners can do replacement
+				provisionerMap[noExpirationValue] = test.Provisioner()
+				provisionerMap[noExpirationValue].Spec = provisionerMap[expirationValue].Spec
+
+				provisionerMap[consolidationValue].Spec.Consolidation = &v1alpha5.Consolidation{Enabled: lo.ToPtr(true)}
+				provisionerMap[emptinessValue].Spec.TTLSecondsAfterEmpty = lo.ToPtr[int64](0)
+				provisionerMap[expirationValue].Spec.TTLSecondsUntilExpired = lo.ToPtr[int64](0)
+				provisionerMap[expirationValue].Spec.Limits = disableProvisioningLimits
+				provisionerMap[driftValue].Spec.Limits = disableProvisioningLimits
+				for _, p := range provisionerMap {
+					env.ExpectCreatedOrUpdated(p)
+				}
+				driftNodeTemplate.Spec.AMIFamily = &v1alpha1.AMIFamilyBottlerocket
+				env.ExpectUpdated(driftNodeTemplate)
+
+				By("waiting for the nodes across all deprovisoiners to get deleted")
+				wg = sync.WaitGroup{}
+				wg.Add(4)
+				go func() {
+					defer GinkgoRecover()
+					defer wg.Done()
+
+					env.MeasureDurationFor(func() {
+						env.EventuallyExpectDeletedNodeCountWithSelector("==", int(float64(nodeCountPerProvisioner)*0.8), labels.SelectorFromSet(map[string]string{
```

Comment:
If you change the factor of .2 to .5, as per my suggestion above make sure to change this to .5 as well.